examples/bare-example/package.json

```diff
@@ -1,13 +1,14 @@
 {
-  "name": "bare-example-notice",
+  "name": "react-native-legal-bare-example",
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "android": "react-native run-android --no-packager",
-    "ios": "react-native run-ios --no-packager",
+    "android": "yarn react-native legal-generate && react-native run-android --no-packager",
```

Comment:
I've updated the `ios` and `android` commands to run `legal-generate` automatically before every build.